src/neo/Persistence/ReadOnlyView.cs

```diff
@@ -19,6 +19,7 @@ public class ReadOnlyView : StoreView
         public override DataCache<SerializableWrapper<uint>, HeaderHashList> HeaderHashList => new StoreDataCache<SerializableWrapper<uint>, HeaderHashList>(store, Prefixes.IX_HeaderHashList);
         public override MetaDataCache<HashIndexState> BlockHashIndex => new StoreMetaDataCache<HashIndexState>(store, Prefixes.IX_CurrentBlock);
         public override MetaDataCache<HashIndexState> HeaderHashIndex => new StoreMetaDataCache<HashIndexState>(store, Prefixes.IX_CurrentHeader);
+        public override MetaDataCache<GlobalVariableState> GlobalCounter => new StoreMetaDataCache<GlobalVariableState>(store, Prefixes.IX_CurrentContractID);
```

Comment:
Something related to SmartContract?

Reply:
yeap.It stores a global contract counter, and can store some other things in the future